src/AutoSubscriptions.ts

```diff
@@ -213,7 +213,7 @@ export const AutoSubscribeStore: ClassDecorator = <TFunction extends Function>(f
 };
 
 // Triggers the handler of the most recent @enableAutoSubscribe method called up the call stack.
-function makeAutoSubscribeDecorator(shallow = false, defaultKeyValues: string[]): MethodDecorator {
+function makeAutoSubscribeDecorator(shallow = false, defaultKeyValues?: string[]): MethodDecorator {
```

Comment:
Suggestion: Rename `defaultKeyValues` since these are always used, not just "defaults". Something like `endKeyValues`?